Hackerspaces/PrefPaneTableViewController.swift

```diff
@@ -15,18 +15,40 @@ class PrefPaneTableViewController: UITableViewController {
             toggle.isOn = SharedData.isInDebugMode()
         }
     }
+
     @IBAction func toggleDebug(_ sender: UISwitch) {
 
         SharedData.toggleDebugMode()
         toggle.isOn = SharedData.isInDebugMode()
         if toggle.isOn {
-            let alert = UIAlertController(title: "Debug mode enabled", message: "debug mode allows you to access error details when a hackerspace is unreachable", preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
-            present(alert, animated: true, completion: nil)
+            let alert = UIAlertController(title: "Advanced mode enabled", message: "Advanced mode displays more advanced features useful for hackerspace API developers", preferredStyle: .alert)
```

Comment:
There are two new alerts in this commit, maybe it's useful to abstract them in their own function